rocker-compiler/src/main/antlr4/com/fizzed/rocker/antlr4/RockerParser.g4

```diff
@@ -59,7 +59,7 @@ templateContent
     ;
 
 block
-    :   (ifBlock | forBlock | withBlock|switchBlock)
+    :   (ifBlock | forBlock | withBlock|switchBlock|switchExpressionBlock)
```

Comment:
Can you add spaces between the | chars to match the style

Reply:
done